data/schema.json

```diff
@@ -2420,8 +2420,8 @@
               "deprecationReason": null
             },
             {
-              "name": "is_contactable",
-              "description": "Are we able to display a contact form on artwork pages?",
+              "name": "is_inquireable",
```

Comment:
Yup! It’s a representation of `schema.graphql` that can be used by the `babel-relay-plugin` to verify that a compiled Relay query matches existing fields and how these need to be mapped into objects.